src/Simplex/Chat.hs

```diff
@@ -3589,15 +3590,14 @@ processAgentMessageConn vr user@User {userId} corrId agentConnId agentMessage =
           -- TODO only acknowledge without saving message?
           -- probably this branch is never executed, so there should be no reason
           -- to save message if contact hasn't been created yet - chat item isn't created anyway
-          withAckMessage agentConnId conn meta False $ \cmdId -> do
-            (_conn', _) <- saveDirectRcvMSG conn meta cmdId msgBody
-            pure False
+          withAckMessage' agentConnId meta $ do
+            (_conn', _) <- saveDirectRcvMSG conn meta msgBody
+            pure ()
```

Comment:
`void` ?

Reply:
yeah.. I just wanted to keep conn' to show it's changed here as in other places. Probably this whole logic can be removed (only ack without saving message - see comment above)

Reply:
changed to void